Dockerfile

```diff
@@ -21,4 +19,10 @@ RUN             apk add --virtual .build-deps \
                 python -c 'import crcmod._crcfunext' && \
                 apk add --virtual .run-deps \
                     ca-certificates && \
-                apk del .build-deps
+                apk del .build-deps && \
+                wget https://github.com/jwilder/dockerize/releases/download/${DOCKERIZE_VERSION}/dockerize-alpine-linux-amd64-${DOCKERIZE_VERSION}.tar.gz && \
+                tar -C /usr/local/bin -xzvf dockerize-alpine-linux-amd64-${DOCKERIZE_VERSION}.tar.gz && \
+                rm dockerize-alpine-linux-amd64-${DOCKERIZE_VERSION}.tar.gz
+
+ENTRYPOINT      ["dockerize", "-template", "/root/.boto.tmpl:/root/.boto", "gsutil"]
+CMD             ["version", "-l"]
```

Comment:
These should probably move back to the original position since it is unlikely to change